pychromecast/socket_client.py

```diff
@@ -809,4 +891,10 @@ def tear_down(self):
         """ Called when controller is destroyed. """
         super(ReceiverController, self).tear_down()
 
+        self.status = None
+        self.launch_failure = None
+        self.app_to_launch = None
```

Comment:
Ah, I see now that not all code is included in this PR but some have been committed. Nevermind.